src/drivers/intel/cavs/mn.c

```diff
@@ -55,7 +55,7 @@ int mn_set_mclk(uint16_t mclk_id, uint32_t mclk_rate)
 	int clk_index = -1;
 	int ret = 0;
 
-	if (mclk_id > 1) {
+	if (mclk_id >= DAI_NUM_SSP_MCLK) {
```

Comment:
@jajanusz this change doesnt seem quite intuitive. You're changing from a value of 1 to a platform-specific value of 2 for all platforms?

Reply:
I just don't want to have hardcoded value `1` here. We can now have it as f.e. `DAI_MCLK_MAX_ID=1` or `DAI_NUM_SSP_MCLK=2`, that is more similar to other values in dai.h, there are all NUMs, and having one MAX_ID would be inconsistent. Another problem here is that I have to define it in many places to have the same value `2`, alternatively I can create new file in intel/cavs/.../dai.h just for 1 define and include it from platform-specific DAIs, but it's even weirder to do that for 1 symbol.

However I should change also that hardcoded `1` in next line.

Reply:
Did I give ok arguments behind it or you still don't like this change? ;)

Reply:
fine with me

Reply:
@jajanusz yes, the change itself looks fine. I was just arguing about the commit message not conveying the intention properly.